proto/model_deployment_service.proto

```diff
@@ -0,0 +1,181 @@
+// Proto definition for the deployment service.
```

Comment:
Could we make a JIRA task for the submoduling the scalabel?

Reply:
sure, I was actually thinking it makes more sense to submodule the proto files maybe? since they will be a mutual dependency.